### PR TITLE
Feat submit forms

### PR DESCRIPTION
**Pull request by mixcas**
I think it needs more styling, can u check @varffs ?

### CODE REVIEW

File: page-submit.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    $open_submissions = IGV_get_option('_igv_submit_options', '_igv_open_submissions');
+
+    if (!empty($open_submissions)) {
+      foreach($open_submissions as $index => $submission) {
+  ?>
+  <article <?php post_class('container submission'); ?>>
+    <div class="grid-row margin-top-basic margin-bottom-basic u-flex-center">
+      <div class="grid-item item-s-6">
+        <?php echo wp_get_attachment_image($submission['_igv_submission_image_id'], 'l-6'); ?>
```
Comment (varffs) [inline page-submit.php]: need to do conditional checks to see if these values exist

File: page-submit.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      <div class="grid-item item-s-8 margin-bottom-basic">
+        <?php echo apply_filters('the_content', $submission['_igv_submission_desc']); ?>
+      </div>
+    </div>
+    <div class="grid-row u-flex-center">
+      <div class="grid-item item-s-8 margin-bottom-basic">
+      <div class="text-align-center"><a class="expandable-toggle no-hide link-button" data-exapandable-id="expandable-<?php echo $index; ?>"><?php echo $submission['_igv_submission_button_text']; ?></a></div>
+      </div>
+    </div>
+    <div id="expandable-<?php echo $index; ?>" class="expandable-content grid-row u-flex-center">
```
Comment (varffs) [inline page-submit.php]: I dont think using the exact same expandable codebase is the right approach. As there are other layouts with this drawer UX like the About page. So either a different implementation for drawers as opposed to expandable news items, or just custom implementation for the relevant templates seems better to me

Comment (mixcas) [inline page-submit.php]: Why? What are the differences?

Comment (varffs) [inline page-submit.php]: Basically to do with what the toggling item does or does not. Could build onto the existing codebase but the current implementation smells bad, just looks like backwards hacking to fix a different usecase

Comment (mixcas) [inline page-submit.php]: kk, i have an idea

**Comment by mixcas** [comment]
@varffs k, u can take over and fix styles as needed. the design has a diff color assigned to the second open submission, i was wondering if that's custom or simple nouns/pairs `nth-of-type` styling.

**Comment by varffs** [comment]
This is styled now including the errors. 

However we need to workout the submission. If you submit an unvalid form it isn't clear as the page reloads with the form (and its validation notifications) hidden by default

**Pull request merged**